sources/osg/Shape.js

```diff
@@ -330,9 +330,9 @@ define( [
         var vertexes = new MACROUTILS.Float32Array( [ -1.0, -1.0, -1.0, 4.0, 4.0, -1.0 ] );
 
         var indexes = new MACROUTILS.Uint16Array( 3 );
-        indexes[ 0 ] = 0;
+        indexes[ 0 ] = 2;
```

Comment:
We want to reduce the number of fragment in composer, making sure tri si drawn fullscreen once, and do it as early  as possible, for all 2D fullscreen passes